js/digifinex.js

```diff
@@ -0,0 +1,528 @@
+            'datetime': this.iso8601 (timestamp),
+            'lastTradeTimestamp': parseInt (order['finished_date']) * 1000,
+            'symbol': symbol,
+            'type': undefined,
+            'side': side,
+            'price': this.safeFloat (order, 'price'),
+            'average': this.safeFloat (order, 'avg_price'),
+            'amount': this.safeFloat (order, 'amount'),
+            'remaining': this.safeFloat (order, 'amount') - this.safeFloat (order, 'executed_amount'),
+            'filled': this.safeFloat (order, 'executed_amount'),
+            'status': status,
+            'fee': undefined,
+        };
+        return result;
+    }
+
+    async fetchOpenOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        // exchange-specific non-unified parameters: page, type
+        // page: Page Num, page=1 for 1st page. None for 1st page by default.
+        // type：buy/sell/buy_market/sell_market，none for all types
+        let request = {
+            'symbol': symbol,
+        };
+        let response = await this.privateGetOpenOrders (this.extend (request, params));
+        let orders = this.parseOrders (response['orders'], undefined, since, limit);
+        return orders;
+    }
+
+    async fetchClosedOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        // Only supports historical orders of last 3 days
+        // `since` has only date precision in digifinex
+        // exchange-specific non-unified parameters: page, type
+        // page: Page Num, page=1 for 1st page. None for 1st page by default.
+        // type：buy/sell/buy_market/sell_market，none for all types
+        let request = {
+            'symbol': symbol,
+        };
+        if (since) {
+            request['date'] = this.dateUTC8 (since);
+        }
+        let response = await this.privateGetOrderHistory (params);
+        let filtered = this.parseOrders (response['orders'], undefined, since, limit);
+        return filtered;
+    }
+
+    async fetchOrder (id, symbol = undefined, params = {}) {
+        let response = await this.privateGetOrderInfo (this.extend ({
+            'order_id': parseInt (id),
+        }, params));
+        return this.parseOrder (response);
+    }
+
+    parseOHLCV (ohlcv, market = undefined, timeframe = '1m', since = undefined, limit = undefined) {
+        return [
+            ohlcv[0],
+            ohlcv[5],
+            ohlcv[3],
+            ohlcv[4],
+            ohlcv[2],
+            ohlcv[1],
+        ];
+    }
+
+    async fetchOHLCV (symbol, timeframe = '1m', since = undefined, limit = undefined, params = {}) {
+        let request = {
+            'symbol': symbol,
+            'type': 'kline_' + timeframe,
+        };
+        let response = await this.privateGetKline (this.extend (request, params));
+        return this.parseOHLCVs (response['data'], undefined, timeframe, since, limit);
+    }
+
+    nonce () {
```

Comment:
We don't need to redefine the `nonce` method, if we are not calling it anywhere in this file.